### PR TITLE
Generate logger local in RestResource (+fixes tests)

### PR DESCRIPTION
**Pull request by Florian-Schoenherr**

### CODE REVIEW

File: src/main/resources/codegen/RestResource.ftl [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package ${packageName};
 
 import io.vlingo.actors.Stage;
+import io.vlingo.actors.Logger;
 import io.vlingo.http.resource.Resource;
 import io.vlingo.http.resource.ResourceHandler;
```
Comment (VaughnVernon) [inline src/main/resources/codegen/RestResource.ftl:6]: Remove.

File: src/main/resources/codegen/RestResource.ftl [hunk 2]
```diff
@@ -26,12 +27,14 @@ public class ${resourceName} extends ResourceHandler {
 </#if>
```
Comment (VaughnVernon) [inline src/main/resources/codegen/RestResource.ftl:27]: Remove `extends ResourceHandler ` from both if and else declarations.

**Review by VaughnVernon — CHANGES_REQUESTED** [review]
@Florian-Schoenherr Nice work. Please remove the unnecessary import of `ResourceHandler` and both `extends ResourceHandler ` uses in class declarations.

**Comment by VaughnVernon** [comment]
@Florian-Schoenherr You should be able to test by running:

`$ mvn clean test`
or
`$ mvn clean install`

**Comment by Florian-Schoenherr** [comment]
Tests work again, thanks to @danilo-ambrosio!

I tried removing `extends ResourceHandler` on schemata as a test, which then gives back 404 on every API request. ServerActor also doesn't log routes from Resources without `extends ResourceHandler`.

**Review by VaughnVernon — CHANGES_REQUESTED** [review]
@Florian-Schoenherr Why is MySQL being used and replacing Postgres? If you prefer to use MySQL locally for your own testing, that's fine. But I am not sure why that would be replaced for everyone. Postgres is sort of our "standard" relational database.

If you and Danilo have agreed to this for some reason let me know. Otherwise I'd like for these Postgres to MySQL changes removed.

**Comment by VaughnVernon** [comment]
@Florian-Schoenherr @danilo-ambrosio Sorry, I was unaware of any agreement to switch from Postgres to MySQL. I am fine with it if it is needed for some specific reason. Just let me know and I can merge all this.

File: src/main/java/io/vlingo/xoom/codegen/template/TemplateStandard.java [hunk 3]
```diff
@@ -56,7 +56,7 @@
                 return Template.REST_RESOURCE_RETRIEVE_METHOD.filename;
             }
 
-            if(parameters.find(REQUIRE_ENTITY_LOADING)) {
+            if((boolean) parameters.find(REQUIRE_ENTITY_LOADING)) {
```
Comment (VaughnVernon) [inline src/main/java/io/vlingo/xoom/codegen/template/TemplateStandard.java]: What happens if the property is missing? Will there be a runtime NPE or CCE?

File: src/main/java/io/vlingo/xoom/codegen/template/TemplateStandard.java [hunk 4]
```diff
@@ -88,7 +88,7 @@
     XOOM_INITIALIZER(templateParameters -> Template.XOOM_INITIALIZER.filename,
             (name, parameters) -> "XoomInitializer"),
 
-    BOOTSTRAP(parameters -> parameters.find(USE_ANNOTATIONS) ?
+    BOOTSTRAP(parameters -> (boolean) parameters.find(USE_ANNOTATIONS) ?
```
Comment (VaughnVernon) [inline src/main/java/io/vlingo/xoom/codegen/template/TemplateStandard.java]: What happens if the property is missing? Will there be a runtime NPE or CCE?

**Review by VaughnVernon — CHANGES_REQUESTED** [review]
@Florian-Schoenherr I am not necessarily requesting a change. I just need to know if these are safe casts, always guaranteed to be non-null boolean.

**Comment by Florian-Schoenherr** [comment]
[MySQL was introduced here](https://github.com/vlingo/vlingo-xoom/commit/479d550cd3473dd015f347fae088765a91ad2adb) by @danilo-ambrosio. I think it's meant for testing, but it is also not replacing postgres right there, but hsqldb. Danilo only told me MySQL will be part of integration testing just for a while.

Casting was maybe not fine, changed it.

**Pull request merged**